doc/code/qml_devices.rst

```diff
@@ -1,7 +1,7 @@
-qml.plugins
+qml.devices
```

Comment:
I know it's a bit late to point this out, but I suddenly wonder if we will be introducing confusion between

* `qml.device`
* `qml.Device`
* `qml.devices`

I suppose only option is to be really clear in the docs